viewer/view/IPCClient.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <QObject>
+#include <QUrl>
+
+namespace hdrv {
+  
+class IPCClient : public QObject
+{
+  Q_OBJECT
+
+public:
+  IPCClient() {}
```

Comment:
As it's a QObject, it should probably be something like, `IPCClient(QObject * parent = nullptr) : QObject(parent) {}`